src/components/Calculator.scss

```diff
@@ -0,0 +1,42 @@
+.App {
+  display: flex;
+  justify-content: center;
+  margin-top: 25%;
+  width: 100vw;

```

Comment:
- [ ] Kindly make sure your design is at the center of the webpage. It's currently almost at the bottom of the webpage. Kindly fix this 🙏 

![image](https://user-images.githubusercontent.com/74814780/199839041-24f0b18a-046e-416c-8bc3-46cc18a8e983.png)